msteams-platform/bots/build-a-bot.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Let's build a bot
+description: Learn to build a basic chatbot using Teams Toolkit, Bot Framework SDK. Additionally, learn to build an AI bot.
+ms.localizationpriority: high
+ms.date: 09/25/2024
+ms.topic: conceptual
+ms.author: surbhigupta
+---
+
+# Let's build a bot
```

Comment:
This is a necessary article and a good start, but I feel like we need more content here and explanatory guidance for it to feel like a complete article.

Reply:
@erikadoyle I've added information about bot basics. I'll try to find additional information about building a bot.